frame/bounties/src/lib.rs

```diff
@@ -131,11 +136,31 @@ pub trait Config: frame_system::Config + pallet_treasury::Config {
 
 	/// Weight information for extrinsics in this pallet.
 	type WeightInfo: WeightInfo;
+
+	/// Maximum number of subbounty that can be added to active bounty.
+	type MaxSubBountyCount: Get<u32>;
 }
 
 /// An index of a bounty. Just a `u32`.
 pub type BountyIndex = u32;
 
+/// A Subbounty proposal.
+#[derive(Encode, Decode, Clone, PartialEq, Eq, RuntimeDebug)]
+pub struct SubBounty<AccountId, Balance, BlockNumber> {
+	/// The account proposing it.
+	proposer: AccountId,
```

Comment:
why need this?

Reply:
Indeed not required, will remove it. Got added based on the assumption that possibility of subbounty gets added or proposed by signed origin other than curator.